lib/helpers.js

```diff
@@ -60,6 +60,24 @@ exports.checkPropertyName = function(name) {
 };
 
 /**
+ * Validate remote method name. Allows `.` in method name for prototype
+ * @param {String} name The user input
+ * @returns {String|Boolean}
+ */
+exports.checkRemoteMethodName = function(name) {
```

Comment:
Oh, I see, we already have `checkPropertyName`. It looks to me that one is an exception, as other method names start with `validate`, so I think it's still better to call the new method `validateRemoteMethodName`.